servicepulse/usage.md

```diff
@@ -37,7 +38,7 @@ Detected endpoints are those identified by the system as NServiceBus endpoints a
 
 ### Detected broker queues
 
-This option will not be displayed for non-broker transports (e.g. MSMQ and Azure Storage Queues).
+This option will not be displayed for non-broker transports.
```

Comment:
Why would we not provide an example? We said customers don't know what things are, so this straight away tells them if they fall into this bucket or not.